public/js/comment.js

```diff
@@ -26,4 +26,13 @@ console.log(e.target);
             alert(response.statusText);
           }
     }
+}
+
+$('#delete-btn').click(deleteComment);
+
+async function deleteComment (e) {
+  e.preventDefault();
+
+  const deleteElement = $('.comment-container');
+  deleteElement.remove();
 }
```

Comment:
this needs to fetch the request in the api - currently this is deleting all elements with the class comment-container - you just need to fetch and then refresh the page